db/models/applications.py

```diff
@@ -0,0 +1,182 @@
+import datetime
+import random
+import uuid
+from operator import itemgetter
+
+from db import db
+from db.models.status import Status
+from sqlalchemy import DateTime
+from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.orm.exc import NoResultFound
+from sqlalchemy.sql import func
+from sqlalchemy_utils.types import UUIDType
+
+
+def started_at():
+
+    raw_date = datetime.datetime.now(datetime.timezone.utc)
+    formatted_date = raw_date.strftime("%Y-%m-%d %H:%M:%S")
+    return formatted_date
+
+
+class Applications(db.Model):
+
+    id = db.Column(
+        "id",
+        UUIDType(binary=False),
+        default=uuid.uuid4,
+        primary_key=True,
+        nullable=False,
+    )
+
+    account_id = db.Column("account_id", db.String(), nullable=False)
```

Comment:
would prefer clumping variables like this more closely. Its a bit painful to figure out the logical chunks.

Reply:
have gone through the repo and cleaned up/ clumped up blocks of code so should be clearer now! :D